packages/mendel-config/src/bundle-config.js

```diff
@@ -23,6 +36,20 @@ function BundleConfig(id, options, config) {
         options,
         ['generator', 'outfile', 'outlet', 'entries', 'require', 'external']
     );
+
+    Object.keys(this.options).forEach((optionKey) => {
+        if (supportedOptions.indexOf(optionKey) === -1) {
+            unsupportedOptions.push(optionKey);
+        }
+    });
+
+    if (unsupportedOptions.length > 0) {
+        console.log([
+            '[Config][WARN] Following options are not supported for bundle: ' + id + '.\n',
```

Comment:
Please use `require('debug')('mendel:config')`. This warning is non essential in most scenarios, since you can have YAML pointers to objects of arbitrary names the user might create.